### PR TITLE
DEV-1

### PR DESCRIPTION
**Pull request by SiarheySkuratovich**

### CODE REVIEW

File: Dev_1.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+public class Dev_1 {
+    public static void main(String[] args){
```
Comment (AlexanderSoroka) [inline Dev_1.java]: codestyle violation

File: Dev_1.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        int temp;
```
Comment (AlexanderSoroka) [inline Dev_1.java]: uninitialized local variable

File: Dev_1.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        for(int i = 0; i <= 100; i++) {
```
Comment (AlexanderSoroka) [inline Dev_1.java]: codestyle violation

File: Dev_1.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+            if(i % 3 == 0) {
+                temp = i / 3;
+                System.out.println("3*" + temp);
+            }
```
Comment (AlexanderSoroka) [inline Dev_1.java]: codestyle violation

File: Dev_1.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+public class Dev_1 {
+    public static void main(String[] args){
+        int temp;
+        for(int i = 0; i <= 100; i++) {
+            if(i % 3 == 0) {
+                temp = i / 3;
```
Comment (AlexanderSoroka) [inline Dev_1.java]: you don't actually need this temp variable

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: Dev_1.java (hunk 1 above)
Comment (SiarheySkuratovich) [inline Dev_1.java]: Can you tell me exactly what is wrong?

File: Dev_1.java (hunk 2 above)
Comment (SiarheySkuratovich) [reply Dev_1.java]: Is it unacceptable?

File: Dev_1.java (hunk 3 above)
Comment (SiarheySkuratovich) [inline Dev_1.java]: it's relly hard to find the mistake through the whole style document. Pleeease, give a hint

File: Dev_1.java (hunk 4 above)
Comment (SiarheySkuratovich) [inline Dev_1.java]: Similarly

File: Dev_1.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+public class Dev_1 {
+    public static void main(String[] args) {
```
Comment (AlexanderSoroka) [inline Dev_1.java]: codestyle violation (4.2)

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Pull request merged**